### PR TITLE
[TE] Create Alert Flow Bug Fix

### PR DESCRIPTION
**Pull request by newsummit**
Now preserving pattern and sensitivity selections after alert creation into alert edit UI.

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/manage/alerts/edit/route.js [hunk 1]
```diff
@@ -37,20 +37,28 @@ export default Ember.Route.extend({
...
+     } = model.function;
 
     let metricId = '';
     let metricDataUrl = '';
     let metricDimensionURl = '';
     let selectedAppName = '';
 
+    // Add a parsed properties array to the model
+    const propsArray = alertProps.split(';').map((prop) => {
+      let p = prop.split('=');
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/manage/alerts/edit/route.js]: nit: we could use destructuring here.

File: thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js [hunk 2]
```diff
@@ -717,10 +712,15 @@ export default Ember.Controller.extend({
...
     'selectedDimension',
     'selectedFilters',
+    'selectedPattern',
     'selectedGranularity',
+    'sensitivityWithDefault',
     function() {
       let gkey = '';
+      let mergedProps = {};
       const granularity = this.get('graphConfig.granularity').toLowerCase();
+      const pattern = encodeURIComponent(this.get('selectedPattern'));
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js]: nit: let's use this.getProperties

File: thirdeye/thirdeye-frontend/app/pods/manage/alerts/edit/route.js (hunk 1 above)
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/pods/manage/alerts/edit/route.js]: done

File: thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js (hunk 2 above)
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js]: done

**Pull request merged**